Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41785&attempt_id=77a5ee08-32ea-4487-a97c-8db5be023f79)

Attempted: sequeler

<details><summary>Partial log (click to expand)</summary><p>

```
Installing /build/source/data/assets/icons/128x128/com.github.alecaddd.sequeler.svg to /nix/store/78zpn1z3ppffdpq8hix4lgmhd1716hhj-sequeler-0.5.5/share/icons/hicolor/scalable/apps
Installing /build/source/schemas/com.github.alecaddd.sequeler.gschema.xml to /nix/store/78zpn1z3ppffdpq8hix4lgmhd1716hhj-sequeler-0.5.5/share/glib-2.0/schemas
Running custom install script '/nix/store/rsa1rsd6c6szy0dmi61599b33aqiy269-python3-3.6.5/bin/python3.6m /nix/store/3pihsh6nsn4nwy71hvz1p05fwj3dbk5c-meson-0.46.1/bin/meson --internal gettext install --subdir=po --localedir=share/locale --pkgname=com.github.alecaddd.sequeler'
Running custom install script '/usr/bin/env python3 /build/source/meson/post_install.py'
Failed to run install script '/usr/bin/env python3 /build/source/meson/post_install.py'
FAILED: meson-install
/nix/store/rsa1rsd6c6szy0dmi61599b33aqiy269-python3-3.6.5/bin/python3.6m /nix/store/3pihsh6nsn4nwy71hvz1p05fwj3dbk5c-meson-0.46.1/bin/meson --internal install /build/source/build/meson-private/install.dat
ninja: build stopped: subcommand failed.
builder for '/nix/store/ms4zrx1w5gzl83n7xhq5bcbwjaqld1fj-sequeler-0.5.5.drv' failed with exit code 1
error: build of '/nix/store/ms4zrx1w5gzl83n7xhq5bcbwjaqld1fj-sequeler-0.5.5.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/sequeler/default.nix
+++ b/pkgs/applications/misc/sequeler/default.nix
@@ -1,10 +1,10 @@
 { stdenv, fetchFromGitHub
-, cmake, ninja, pkgconfig, vala, gobjectIntrospection, gettext, wrapGAppsHook
-, gtk3, glib, granite, libgee, libgda, gtksourceview, libxml2 }:
+, meson, ninja, pkgconfig, vala, gobjectIntrospection, gettext, wrapGAppsHook
+, gtk3, glib, granite, libgee, libgda, gtksourceview, libxml2, libsecret }:
 
 
 let
-  version = "0.5.4";
+  version = "0.5.5";
   sqlGda = libgda.override {
     mysqlSupport = true;
     postgresSupport = true;
@@ -17,12 +17,17 @@ in stdenv.mkDerivation rec {
     owner = "Alecaddd";
     repo = "sequeler";
     rev = "v${version}";
-    sha256 = "05c7y6xdyq3h9bn90pbz03jhy9kabmgpxi4zz0i26q0qphljskbx";
+    sha256 = "0jv7nx9k1qw2i3cmg0vnahz4qfam03xypas975x40icqd3bhfgj3";
   };
 
-  nativeBuildInputs = [ cmake ninja pkgconfig vala gobjectIntrospection gettext wrapGAppsHook ];
+  nativeBuildInputs = [ meson ninja pkgconfig vala gobjectIntrospection gettext wrapGAppsHook ];
 
-  buildInputs = [ gtk3 glib granite libgee sqlGda gtksourceview libxml2 ];
+  buildInputs = [ gtk3 glib granite libgee sqlGda gtksourceview libxml2 libsecret ];
+
+  postPatch = ''
+    chmod +x meson/post_install.py
+    patchShebangs meson/post_install.py
+  '';
 
   meta = with stdenv.lib; {
     description = "Friendly SQL Client";

